### PR TITLE
[RM Ch8]: Linewrap to 120 chars

### PR DESCRIPTION
**Pull request by CsatariGergely**
Partially closes #2834

### CODE REVIEW

**Review by ulikleber — APPROVED** [review]

**Review by walterkozlowski — APPROVED** [review]

File: doc/ref_model/chapters/chapter08.rst [hunk 1]
```diff
@@ -213,118 +305,231 @@ Requirements, Reference Architecture & Industry Standards Intersect
...
+|           | Platform Services                                      | Storage                 | Network Services      |
+|           +-------+-------+-------+-------+-------+-------+--------+--------+--------+-------+-------+-------+-------+
+|           | Iden- | Image | Plac- | Comp- | Netw- | Mess- | DB     | Ephem- | Persi- | Pers- | Mana- | Unde- | Over- |
+|           | tity  |       | ement | ute   | orki- | age   | Server | eral   | stent  | iste- | geme- | rlay  | lay   |
+|           |       |       |       |       | ng    | Queue |        |        | Block  | nt    | nt    | (Pro- |       |
+|           |       |       |       |       |       |       |        |        |        | Obje- |       | vid-  |       |
+|           |       |       |       |       |       |       |        |        |        | ct    |       | er)   |       |
++===========+=======+=======+=======+=======+=======+=======+========+========+========+=======+=======+=======+=======+
+| Control   | ✅    | ✅    | ✅    | ✅    | ✅    | ✅    | ✅     |        | ✅     |       | ✅    | ✅    | ✅    |
```
Comment (gkunz) [inline doc/ref_model/chapters/chapter08.rst:523]: looks like the padding is off by one.

File: doc/ref_model/chapters/chapter08.rst [hunk 2]
```diff
@@ -213,118 +305,231 @@ Requirements, Reference Architecture & Industry Standards Intersect
...
+| Nodes     |       |       |       |       |       |       |        |        |        |       |       |       |       |
++-----------+-------+-------+-------+-------+-------+-------+--------+--------+--------+-------+-------+-------+-------+
+| Workload  |       |       |       | ✅    | ✅    |       |        | ✅     | ✅     | ✅    | ✅    | ✅    | ✅    |
```
Comment (gkunz) [inline doc/ref_model/chapters/chapter08.rst:526]: looks like the padding is off by one.

File: doc/ref_model/chapters/chapter08.rst [hunk 3]
```diff
@@ -213,118 +305,231 @@ Requirements, Reference Architecture & Industry Standards Intersect
...
+| Nodes     |       |       |       |       |       |       |        |        |        |       |       |       |       |
+| (Compute) |       |       |       |       |       |       |        |        |        |       |       |       |       |
++-----------+-------+-------+-------+-------+-------+-------+--------+--------+--------+-------+-------+-------+-------+
+| Storage   |       |       |       |       |       |       |        |        | ✅     | ✅    | ✅    | ✅    | ✅    |
```
Comment (gkunz) [inline doc/ref_model/chapters/chapter08.rst:530]: looks like the padding is off by one.

**Review by gkunz — CHANGES_REQUESTED** [review]

File: doc/ref_model/chapters/chapter08.rst (hunk 1 above)
Comment (CsatariGergely) [reply doc/ref_model/chapters/chapter08.rst:523]: It is off only due to the tickmark characters (✅). 
It renders nicely: 
image

**Review by gkunz — APPROVED** [review]

File: doc/ref_model/chapters/chapter08.rst (hunk 1 above)
Comment (gkunz) [reply doc/ref_model/chapters/chapter08.rst:523]: ah, ok. that explains it. Thanks

**Pull request merged**